ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessorTest.kt

```diff
@@ -13,45 +15,131 @@ import java.util.Locale
 @RunWith(AndroidJUnit4::class)
 class VariableProcessorTest {
 
+    private val usLocale = Locale.US
+    private val esLocale = Locale("es", "ES")
```

Comment:
Cool 👍🏻 

Don't have to do it now,  but I guess we're close to adding Spanish strings and testing them? :)

Reply:
I mean, we could potentially run instrumentation tests (that run on an actual device), but I'm not sure it's worth it for this since we need to run them on firebase test lab and are much slower.